src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/UnusedTagsPassUtils.php

```diff
@@ -37,6 +37,17 @@ public static function getDefinedTags(): array
             }
         }
 
+        // get all tags used in PHP configs
+        $files = Finder::create()->files()->name('*.php')->path('Resources')->notPath('Tests')->in(\dirname(__DIR__, 5));
+        foreach ($files as $file) {
+            $contents = file_get_contents($file);
+            if (preg_match_all("{->tag\('([^']+)'}", $contents, $matches)) {
```

Comment:
To be consistent with what is done for XML files, should we also look for tagged iterator arguments ?

Reply:
Yes, will do when I have an example :)